demos/mouse-follow/follow.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"image/color"
+
+	"engo.io/ecs"
+	"engo.io/engo"
+)
+
+type DefaultScene struct{}
+
+type Guy struct {
```

Comment:
Suggestion for later PRs: Move the "Guy" object to demoutils, it's being re-used a bunch.